### PR TITLE
Feat: update klay sdks

### PR DESCRIPTION
**Pull request by ghost**
1. Add specs for:
       - **klay_getBlsInfor**
       - **klay_getActiveAddressFromRegistry** 
       - **klay_getAllRecordsFromRegistry**
       - **klay_getTotalSupply**
2. Update category for **klay_gasPrice,** **klay_gasPriceAt,** **eth_gasPrice**
3. Update description for:
       - **klay_gasPrice**
       - **eth_gasPrice**
       - **klay_maxPriorityFeePerGas**
       - **eth_maxPriorityFeePerGas**
       - **klay_feeHistory**
       - **eth_feeHistory**
4. Update **[Request > params > TracingOptions]**

### CODE REVIEW

File: web3rpc/rpc-specs/paths/klay/miscellaneous/getBlsInfos.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+paths:
+  /klay/miscellaneous/getBlsInfos:
+    post:
+      summary: "[Miscellaneous] klay_getBlsInfos"
+      tags:
+        - klay
+      operationId: getBlsInfos
+      description: |
+        Returns the account and storage values of the specified account including the Merkle-proof. This call can be used to verify that the data you are pulling from is not tampered with.
```
Comment (blukat29) [inline web3rpc/rpc-specs/paths/klay/miscellaneous/getBlsInfos.yaml]: You can find the description in the https://kips.klaytn.foundation/KIPs/kip-113 document. Please copy the description, return value fields, and examples from there.
```suggestion
        Returns all registered BLS public keys and proof-of-possessions of the validators.
```

File: web3rpc/rpc-specs/paths/klay/miscellaneous/getAllRecordsFromRegistry.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+servers:
+  - url: http://localhost:7151
+
+paths:
+  /klay/miscellaneous/getAllRecordsFromRegistry:
+    post:
+      operationId: getAllRecordsFromRegistry
+      summary: "[Miscellaneous] klay_getAllRecordsFromRegistry"
+      description: |
+        **JSONRPC:** `klay_getAllRecordsFromRegistry`
```
Comment (blukat29) [inline web3rpc/rpc-specs/paths/klay/miscellaneous/getAllRecordsFromRegistry.yaml:20]: Please copy the description, parameter names, return value fields, and examples from the https://kips.klaytn.foundation/KIPs/kip-149 document.

```suggestion
		Returns all records of the system contract registered as `name` if it has been registered.

        **JSONRPC:** `klay_getAllRecordsFromRegistry`
```

File: web3rpc/rpc-specs/paths/klay/miscellaneous/getAllRecordsFromRegistry.yaml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                $ref: "../../../components/schemas/common/Common.yaml#/components/schemas/BlockNumberOrHashOrTag"
+          example:  ["kip113", "latest"]
+
+    KlaygetAllRecordsFromRegistryResp:
+      type: object
+      properties:
+        result:
+            type: string
+            description: Returns address of active system contract registered with name if exists.
+            example: "0x0000000000000000000000000000000000000402"
```
Comment (blukat29) [inline web3rpc/rpc-specs/paths/klay/miscellaneous/getAllRecordsFromRegistry.yaml:99]: Please check the actual response from the RPC!
`kip113` doesn't work. `KIP113` works. and the return type is an array of object.

```
> klay.getAllRecordsFromRegistry("KIP113", "latest")
[{
    activation: 0,
    addr: "0x3e80e75975bdb8e04b800485dd28bebec6d97679"
}]
```

File: web3rpc/rpc-specs/paths/klay/miscellaneous/getActiveAddressFromRegistry.yaml [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+servers:
+  - url: http://localhost:7151
+
+paths:
+  /klay/miscellaneous/getActiveAddressFromRegistry:
+    post:
+      operationId: getActiveAddressFromRegistry
+      summary: "[Miscellaneous] klay_getActiveAddressFromRegistry"
+      description: |
+        **JSONRPC:** `klay_getActiveAddressFromRegistry`
```
Comment (blukat29) [inline web3rpc/rpc-specs/paths/klay/miscellaneous/getActiveAddressFromRegistry.yaml:21]: Please copy the description, parameter names, return value fields, and examples from the https://kips.klaytn.foundation/KIPs/kip-149 document.

```suggestion
        Returns the active address of the system contract registered as `name` if exists.

        **JSONRPC:** `klay_getActiveAddressFromRegistry`
```

File: web3rpc/rpc-specs/paths/klay/miscellaneous/getActiveAddressFromRegistry.yaml [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+                $ref: "../../../components/schemas/common/Common.yaml#/components/schemas/BlockNumberOrHashOrTag"
+          example:  ["kip113", "latest"]
+
+    KlaygetActiveAddressFromRegistryResp:
+      type: object
+      properties:
+        result:
+            type: string
+            description: Returns address of active system contract registered with name if exists.
+            example: "0x0000000000000000000000000000000000000402"
```
Comment (blukat29) [inline web3rpc/rpc-specs/paths/klay/miscellaneous/getActiveAddressFromRegistry.yaml:100]: The arguments are case-sensitive. It must be `KIP113` to work.

```suggestion
          example:  ["KIP113", "latest"]

    KlaygetActiveAddressFromRegistryResp:
      type: object
      properties:
        result:
            type: string
            description: Returns address of active system contract registered with name if exists.
            example: "0x3e80e75975bdb8e04b800485dd28bebec6d97679"
```

File: web3rpc/rpc-specs/paths/klay/miscellaneous/getTotalSupply.yaml [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+
+paths:
+  /klay/miscellaneous/getTotalSupply:
+    post:
+      summary: "[Miscellaneous] klay_getTotalSupply"
+      tags:
+        - klay
+      operationId: getTotalSupply
+      description: |
+        Returns the account and storage values of the specified account including the Merkle-proof. This call can be used to verify that the data you are pulling from is not tampered with.
```
Comment (blukat29) [inline web3rpc/rpc-specs/paths/klay/miscellaneous/getTotalSupply.yaml]:
```suggestion
        Returns the native token token supply by tracking block rewards and various burning mechanisms. If some components are not available then a partial result is returned.
```

**Review by blukat29 — APPROVED** [review]
Thanks!

File: web3rpc/sdk/client/java/web3j-ext-test-wrapper/build.gradle [hunk 7]
```diff
@@ -15,7 +15,7 @@ repositories {
 }
 
 dependencies {
-    api "foundation.klaytn:web3rpc-java:v0.9.0"
+    api "foundation.klaytn:web3rpc-java:v1.0.1"
     implementation "org.web3j:core:4.9.7"
```
Comment (ghost) [inline web3rpc/sdk/client/java/web3j-ext-test-wrapper/build.gradle:19]: The generated sdk have version 1.0.1 so i need to change build.gradle version of web3rpc-java to match the local build

File: web3rpc/rpc-specs/paths/klay/miscellaneous/getBlsInfos.yaml [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+                - $ref: "../../../components/requests/JsonRpcRequest.yaml#/components/schemas/JsonRpcRequest"
+                - type: object
+                  properties:
+                    method:
+                      type: string
+                      default: klay_getBlsInfos
+                    params:
+                      type: array
+                      items:
+                        anyOf:
```
Comment (kjeom) [inline web3rpc/rpc-specs/paths/klay/miscellaneous/getBlsInfos.yaml]: @Sotatek-TanHoang As I said, you should avoid anyOf type

File: web3rpc/rpc-specs/paths/klay/miscellaneous/getTotalSupply.yaml [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+                - $ref: "../../../components/requests/JsonRpcRequest.yaml#/components/schemas/JsonRpcRequest"
+                - type: object
+                  properties:
+                    method:
+                      type: string
+                      default: klay_getTotalSupply
+                    params:
+                      type: array
+                      items:
+                        anyOf:
```
Comment (kjeom) [inline web3rpc/rpc-specs/paths/klay/miscellaneous/getTotalSupply.yaml]: Same comments. Please look at above

**Review by kjeom — APPROVED** [review]

**Pull request merged**